attest/verifier/src/status.rs

```diff
@@ -14,36 +14,55 @@
 //! implement that.
 
 use crate::Verify;
-use alloc::{borrow::ToOwned, string::String, vec::Vec};
+use alloc::borrow::ToOwned;
 use mc_attest_core::{
     IasQuoteError, IasQuoteResult, ProductId, SecurityVersion, VerificationReportData,
 };
-use mc_attest_verifier_config::{
-    TrustedMeasurement, TrustedMrEnclaveMeasurement, TrustedMrSignerMeasurement,
+use mc_attest_verifier_config::TrustedMeasurement;
+use mc_attestation_verifier::{
+    Advisories, AdvisoriesVerifier, AdvisoryStatus, TrustedMrEnclaveIdentity,
+    TrustedMrSignerIdentity, Verifier,
 };
 use mc_sgx_core_types::{MrEnclave, MrSigner};
 use mc_sgx_css::Signature;
 use serde::{Deserialize, Serialize};
 
 /// A helper function used to check exceptions to the quote error = fail rule.
-fn check_ids(quote_status: &IasQuoteResult, config_ids: &[String], sw_ids: &[String]) -> bool {
+fn check_ids(quote_status: &IasQuoteResult, advisories: &Advisories) -> bool {
+    let verifier = AdvisoriesVerifier::new(advisories.to_owned()); // TODO: this?
```

Comment:
:wrench: Does this `TODO` still belong here?
```suggestion
    let verifier = AdvisoriesVerifier::new(advisories.to_owned());
```